exercises/dict/dict3.cairo

```diff
@@ -0,0 +1,65 @@
+// dict3.cairo
+// Custom data structure using dicts
+// Using Felt252Dict in structures allow us to modify values stored in memory
```

Comment:
```suggestion
// Using Felt252Dict in structs allow us to simulate mutable data structures
```

We don't really modify the underlying memory. It's just a convenient, high-level way of changing values associated to variables